src/Model/ModelManager.php

```diff
@@ -13,6 +13,7 @@
 
 namespace Sonata\DoctrineMongoDBAdminBundle\Model;
 
+use Doctrine\Common\Collections\ArrayCollection;
```

Comment:
I think this one is deprecated, you should not use common namespace

Reply:
I thought the same, but looks like [not yet](https://github.com/doctrine/collections/issues/193) for `doctrine/collections`.